pyslackersweb/__init__.py

```diff
@@ -52,11 +53,14 @@
         POSTGRESQL_DSN=settings.POSTGRESQL_DSN,
     )
 
-    app.cleanup_ctx.extend([apscheduler, client_session, redis_pool, postgresql_pool])
+    if not skip_scheduler:
```

Comment:
This seems like it should be an application setting instead, allowing everything but start/stop to be called in the apscheduler cleanup context. This lets us ensure everything sets up correctly, even if we don’t run it, and to not make nested apps have to concern themselves with something that’s irrelevant to them.

Reply:
Ah good idea :+1: